doc/Language/variables.pod6

```diff
@@ -498,6 +399,212 @@ augment slang Regex {  # derive from $~Regex and then modify $~Regex
+variable C<$num>. The assignment expression involving the item assignment to the
+Scalar variable C<$num> is thus evaluated first. It returns the assigned value
+C<42>, which in turn forms part of the List C<(42, "str")> constructed by the
+comma operator that also has a higher precedence than the list
+assignment. Finally, the List C<(42, "str")> is list-assigned to C<@array>:
+
+    #`[this statement:] my @array = my $num = 42, "str";
+    # is parsed as      my @array = ( (my $num = 42), "str" )
+    say @array.raku;    # OUTPUT: «[42, "str"]␤» (an Array)
+    say $num.raku;      # OUTPUT: «42␤» (an Int)
+
+Here's a variant:
+
+    #`[this:]      my @foo = (my $bar) = 42, "str";
+    # parsed as:   my @foo = ( $bar = (42, "str") )
+    say $bar.raku;      # OUTPUT: «$(42, "str")␤» (a List)#
+    say @foo.raku;      # OUTPUT: «[(42, "str"),]␤» (an Array)
+
+In this case, the list contextualizer C<( )> puts C<$bar> in a list context, and
+thus triggers a list assignment to the Scalar variable C<$bar>. This means that
+there are two chained list assignments, both having a lower precedence than the
+comma operator C<,> that constructs the List C<(42, "str")>. Due to their right
+associativity, the list assignment expression that is evaluated first is the
+assignment to C<$bar>, which returns the assigned value C<$(42, "str")>, i.e. a
+Scalar containing a two-element List. This value is in turn list-assigned to
+C<@array>, such that it becomes a Array with a single element, namely a List.
+
+See L<operators|/language/operators> for more details on precedence and
+associativity.  For a more elaborate discussion of list assignment in the
+context of variable declarations, see the section on L<declaring a list of
+variables with lexical or package
+scope|/language/variables#index-entry-declaring_a_list_of_variables>.
+
+With list assignment, you can have multiple groups of variables on the left hand
+side and multiple groups of values on the right hand side – but the groupings on
+the left hand side do not impact the assignments.
+
+    #`[this line is] (my ($a, $b), my $c) = (1, (2, 3), 4);
+    # the same as:    my ($a, $b, $c)     = (1, (2, 3), 4);
+    say $a; say $b; say $c;         # OUTPUT: «1␤(2, 3)␤4␤»
+
+In order for the groupings on the left to matter, you need Raku's third method of
+assignment, destructuring assignment.
+
+
+=head2 Destructuring assignment
+
+Destructuring assignment allows you to destructure a (potentially nested) group
+of values on the right hand side into a (potentially nested) group of variables
+on the right hand side. Like list assignment, you trigger destructuring
+assignment by having anything other than a Scalar on the left hand side, but
+then follow the LHS by C<:=> rather than C<=>.  Using C<:=> causes the right
+hand side values to be destructured into the LHS variables using the same syntax
+that is used for L<destructuring
+arguments|/type/Signature#Destructuring_arguments> in function signatures.
+
```

Comment:
OK, but `:=` is a binding assignment. So maybe point to its definition, and indicate this is essentially another way of binding?